src/rules/noAnyRule.ts

```diff
@@ -33,7 +33,7 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static FAILURE_STRING = "Type declaration of 'any' is forbidden";
+    public static FAILURE_STRING = "Type declaration of 'any' is forbidden (use the empty type '{}'?)";
```

Comment:
I think users will still stumble here. Linking to the doc would be best (see other comment)

Wordsmith attempt:
`Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type, the empty type ('{}'), or suppress this occurrence`

Separately, it would be nice if the end of a tslint failure report pointed you to the docs for suppression https://palantir.github.io/tslint/usage/rule-flags/ - nearly every user I've coached was unaware of those.